src/app/search/search-filters/known-values.tsx

```diff
@@ -240,6 +241,12 @@ const knownValuesFilters: FilterDefinition[] = [
       }
     },
   },
+  {
+    keywords: 'focusable',
+    description: tl('Filter.Focusable'),
+    destinyVersion: 2,
+    filter: () => (item) => Object.values(focusingOutputs).includes(item.hash),
```

Comment:
Rather than computing `Object.values(focusingOutputs)` for every item, you could do it once:
```suggestion
    filter: () => {
      const outputValues = Object.values(focusingOutputs);
      return (item) => outputValues.includes(item.hash);
    },
```

Reply:
Good catch!